spec/sanity/events.js

```diff
@@ -13,8 +13,10 @@ describe('logs & events page', function() {
         browser.sleep(500);
     }
 
-    beforeEach(function(){
-        events.route();
+    beforeEach(function(done){
+        components.ui.LoginPage.goTo().login('admin', 'admin');
+        components.ui.layout.goToLogsEvents();
+        browser.sleep(1000).then(done);
```

Comment:
Seem very random .. perhaps a 'browser.waitForAngular()' is more appropriate + waiting for debounce.